Rails-Girls-Summer-Of-Code-2017.md

```diff
@@ -0,0 +1,42 @@
+----------
+
+# Team 276linesofCode - Rails Girls Summer of Code 2017
+
+Hey all,
+
+It has been a long long ride since February and its finally July!
+This is when we start our Rails Girls Summer of Code (RGSoC) 2017 program, by working and contributing for Tessel. Having seen Tessel as a project in the RGSoC project list of 2017, we were eager to take it up at once since, we felt that through this project we would be able to discover both the horizons - Hardware and Software.
```

Comment:
So I guess we'll add the RGSoC intro in the beginning. Will make it more Audience friendly